problems/4.arrays.js

```diff
@@ -2,25 +2,33 @@
 * @param {number[]} arr - The input array
 * @returns {number} - The average of the first and last values
 */
-function firstAndLastAverage() {
-
-}
+const firstAndLastAverage = (arr) => {}
 
 /** Duplicates all values inside an array.
 * Sample input: [1,4,2,5]
 * Sample output: [1,4,2,5,1,4,2,5]
 * @param {number[]} arr - The input array
 * @returns {number[]} - The duplicated array
 */
-function duplicateArray() {
-
+const duplicateArray = (arr) => {
+  a = [];
+  b = [];
+  for (i =0; i < a.length; i++){
```

Comment:
Your array is called arr. Must loop through that twice and push in the elements into an output array. Or look at concat on MDN.